Assets/Scripts/GameState/LeaderboardData.cs

```diff
@@ -56,58 +75,54 @@ public static void SaveScores(Highscores scores)
         stream.Close();
     }
 
-    public static Highscores LoadScores()
+    /// <summary>
+    /// Load all saved scores
+    /// </summary>
+    /// <returns>Returns a ordered list of Highscore structs</returns>
```

Comment:
It's redundant to use the word "returns", as it's already in the markup 😛 (Also added some suggestions on grammar and additional markup.)
```suggestion
    /// <returns>An ordered list of <c>Highscore</c> structs.</returns>
```